core_types/Cargo.toml

```diff
@@ -18,6 +18,9 @@ objekt="0.1.1"
 holochain_core_types_derive = { path = "../core_types_derive" }
 uuid = { version = "0.7", features = ["v4"] }
 regex = "^1.1"
+chrono = "0.4"
+guid-create = "^0.1.1"
```

Comment:
unneccessary dep?

Reply:
oh yes left over from previous, will remove that